testsuite/run_sets/build_validation_add_custom_repositories.yml

```diff
@@ -3,7 +3,7 @@
 # This yaml file contains all the features performing the creation of custom channels
 # for each Maintenance Update
 
-## Add MU repositories BEGIN ###
+## Add Custom repositories BEGIN ###
```

Comment:
ok, i have very sheldonish remark :-)

any reason for the upper case ? ;-)

Reply:
rf sorry, I didn't saw it and merged already the other one.
Will fix in a future one.

Reply:
ok, no big deal @srbarrios